Maven.gitignore

```diff
@@ -7,3 +7,6 @@ release.properties
 dependency-reduced-pom.xml
 buildNumber.properties
 .mvn/timing.properties
+
+# Exclude maven wrapper
+!/.mvn/wrapper/maven-wrapper.jar
```

Comment:
> Usually, one would also add the `Java.gitignore` file to a Java project, which unfortunately contains such a catch-all rule for `.jar`s. 

Thanks, this clears things up a bit. Not having enough context on how projects manage this, I have a couple of general questions (I don't expect to address them as part of this PR):

 - Is it common for people to use `Java.gitignore` without `Maven.gitignore`?
 - Should all the Maven rules from `Java.gitignore` live in `Maven.gitignore`?
 - Is Maven a tool rather than a programming language? If so, it should live under the `Global` namespace...